src/main/java/seedu/address/logic/parser/AddProcCommandParser.java

```diff
@@ -38,16 +39,17 @@ public AddProcCommand parse(String args) throws ParseException {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddProcCommand.MESSAGE_USAGE), pe);
         }
 
-        if (!arePrefixesPresent(argMultimap, PREFIX_INFORMATION, PREFIX_COST, PREFIX_DATE)
-                || !argMultimap.getPreamble().isEmpty()) {
+        if (!arePrefixesPresent(argMultimap, PREFIX_INFORMATION, PREFIX_COST, PREFIX_DATE)) {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddProcCommand.MESSAGE_USAGE));
         }
 
         Information information = ParserUtil.parseInformation(argMultimap.getValue(PREFIX_INFORMATION).get());
         Cost cost = ParserUtil.parseCost(argMultimap.getValue(PREFIX_COST).get());
         Date date = ParserUtil.parseDate(argMultimap.getValue(PREFIX_DATE).get());
+        // New Procedure is not completed by default
+        Completion completion = new Completion("FALSE");
```

Comment:
Nice coding style thus far. I am not sure if the new commit has revised this. However, new Completion will not take in "FALSE". It only takes in "false" in the newer pull requests, so it might be good to tweak this to "false" instead.